setup.cfg

```diff
@@ -0,0 +1,136 @@
+# Global options:
+
+[mypy]
+ignore_missing_imports = True
+exclude = (infogami|scripts/legacy/marc|vendor*)/$
```

Comment:
@dhruvmanila How do we add your excludes with these excludes?

Reply:
```
exclude = (infogami|scripts/legacy/marc|vendor*|scripts/\d+)/$
```
This should work. Let me take a look at why `infogami` is being checked even though it is included in the exclude option. It might be that the regex is incorrect or something related to it being a symlink.

Reply:
Btw, you could compress all those lines into one to ignore the entire `infogami` submodule:
```ini
[mypy-infogami.*]
ignore_errors = True
```